db/migrate/20170725143941_create_api_consumers.rb

```diff
@@ -0,0 +1,12 @@
+class CreateApiConsumers < ActiveRecord::Migration
+  def change
+    create_table :api_consumers do |t|
+      t.string :name
+      t.string :email, null: false, index: true
+      t.string :provider, null: false
+      t.string :uid, null: false, index: true
```

Comment:
@coffeejunk if you look at the design 

https://docs.google.com/document/d/1LcZD8dJXeDnOrYtdrEZ6Dq3820mi2P0FUZNJL6Jd-Dg/edit#heading=h.9eb1b9xngsnr

the OpenIDC information is attribute (or a set of attributes) to the API customer. So if there is a OpenIDC id field you need to save that should be in another OpenIDC specfic field, that way we could add other means of authentication in the future

Also, Google is an OpenIDC provider but we would like it to work with other OpenIDC providers if possible, that does not mean that we implement the sign up for other providers now, but at least make it enough generic